nbs/03_matcher.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def _find_structural_matches(graph: DiGraph, pattern: DiGraph) -&gt; Tuple[DiGraph, dict[NodeName, NodeName]]:
</pre><p>something looks wrong here. Why id there 2 function def lines?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Fixed it</p><p> </p>